src/mainwin.cc

```diff
@@ -2183,7 +2183,8 @@ void MainWindow::actionExport(FileFormat format, const char *type_name, const ch
 	exportFileByName(this->root_geom, format,
 		export_filename.toLocal8Bit().constData(),
 		export_filename.toUtf8());
-	PRINTB("%s export finished.", type_name);
+	PRINTB("%s export finished: %s",
+		type_name % export_filename.toLocal8Bit().constData());
```

Comment:
Since we're printing to the GUI console, should this be `toUtf8()` ?

Reply:
Yes, I think you're right. I just changed it. Both seem to work the same on my system (modern Linux), but maybe on other systems Qt wants utf-8 even if the local codec is something else?

Note that there's some inconsist use of the two in this file (maybe others). For example there's a `PRINTB` in `MainWindow::actionExportCSG()` that uses `toLocal8Bit()`.

Reply:
To my understanding, local 8 bit encoding is for performing system calls outside of Qt. Qt should prefer its own encoding for all string uses. OpenSCAD APIs prefer `std::strings` where possible and UTF-8 otherwise. To my knowledge Linux uses utf-8 as its native 8-bit encoding, while e.g. Windows may use some internal codepage stuff.

Reply:
There may be some inconsistency. If you find any, feel free to fix and PR :)

Reply:
Using utf-8 for most everything internally, then local 8-bit for system interactions makes sense. Btw, I see that as of Qt 5, there's a [QString::toStdString()](http://doc.qt.io/qt-5/qstring.html#toStdString) method. This might be best to use in most places - it does use toUtf8() behind the scenes though.

Reply:
I agree that `toStdString()` would be preferable once we kill Qt4 support.